### PR TITLE
Changelog for beta 2.

### PR DESCRIPTION
**Pull request by vpetrovykh**
This is still a draft, but feel free to comment on things.

I'll flesh out more actual examples of the added features in the next iteration.

### CODE REVIEW

**Comment by 1st1** [comment]
@colinhacks Colin, here's the changelog, you can use it for your b2 blog post

File: docs/changelog/1_0_b2.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+more verbose command options. This instance name can be used in CLI as
+well as programmatically when connecting to EdgeDB.
+
+
+Bindings
+========
+
+* Implement `RFC 1004 <robust_>`_ features for `edgedb-go
+  <https://github.com/edgedb/edgedb-go>`_ driver.
+* Update the `edgedb-python
```
Comment (1st1) [inline docs/changelog/1_0_b2.rst:96]: We also implemented this https://github.com/edgedb/edgedb-python/commit/d5597de15f5c8c3b6d9799bf46afe21abf1bd531 for edgedb-python and edgedb-js

File: docs/changelog/1_0_b2.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  <https://github.com/edgedb/edgedb-python>`_ driver to v0.14.0.
+* Update the `edgedb-js <https://github.com/edgedb/edgedb-js>`_ driver
+  to v0.14.0.
```
Comment (1st1) [inline docs/changelog/1_0_b2.rst:99]: I think we should also mention Deno support. @jaclarke can you contribute a short hello world 10 lines example to mention that properly?

File: docs/changelog/1_0_b2.rst (hunk 1 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_b2.rst:179]: Added a mention of that, too.

File: docs/changelog/1_0_b2.rst (hunk 2 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_b2.rst:182]: Added the release of Deno to the list.

File: docs/changelog/1_0_b2.rst [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+* Release `Deno <https://github.com/edgedb/edgedb-deno>`_ driver.
```
Comment (1st1) [inline docs/changelog/1_0_b2.rst:183]:
```suggestion
* Release the `Deno <https://github.com/edgedb/edgedb-deno>`_ driver.
```

**Review by 1st1 — APPROVED** [review]
Let's merge this

**Pull request merged**